### DISCUSSION TITLE
Using Cesium reality models in iTwin.js

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2946] [Q&A]
Our application is currently working with Reality Models that are stored in ContextShare, uploaded there using mostly Bentley Descartes in 3sm format (and in few cases using iTwin Synchronizer).  Our client has sent some models in Cesium format and asked us to display in our iTwin app.  We only know of using Descartes to upload data into Context and 'File->Import' in Descartes doesn't seem to have a 3D tile option or attach a 3D tile model to a dgn.

Is there any guidance on how we can bring Cesium models into our iTwin app. 

Thanks,

Ovais

**Comment by rbois-bentley** [comment]
Hi, you can try to attach the 3DTile in Descartes using the Reality Mesh dialog, that is the same dialog used to attach a local 3sm format.

**Comment by MarcBedard8** [comment]
You can also get more information on tool to upload different format to Context Share on this forum: https://communities.bentley.com/products/3d_imaging_and_point_cloud_software/w/wiki/55907/upload-reality-mesh-from-orbit-3dm-import-and-upload-tool-to-projectwise-contextshare
